recipes/transannot/meta.yaml

```
@@ -0,0 +1,52 @@
+{% set version = "03cc4a7" %}
+{% set sha256 = "432f31744683fa467a1ab76493818f52d3cb8f87d489c7190c255f18c03d9418" %}
+
+package:
+  name: transannot
+  version: {{ version|replace("-", ".")  }}
+
+build:
+  number: 0
+  run_exports:
+    - {{ pin_subpackage('transannot', max_pin="x") }}
```

```suggestion
    - {{ pin_subpackage('transannot', max_pin=None) }}
```
I don't think this recipe would support SemVer while using Git SHA as a version